### PR TITLE
2.34 update

### PR DESCRIPTION
**Pull request by eljeffeg**

### CODE REVIEW

**Comment by zachriggle** [comment]
@Arusekk mind merging this? Looks good to me

File: macos/install_all.sh [hunk 1]
```diff
@@ -2,5 +2,5 @@
 source ../common/arch.sh
 
 for arch in $ARCHES; do
-brew install binutils-$arch.rb
-done
\ No newline at end of file
+brew upgrade binutils-$arch.rb
```
Comment (chuanchan1116) [inline macos/install_all.sh]: Homebrew does not install formulae with `upgrade` command
```suggestion
brew install binutils-$arch.rb
```

File: generic/install.sh [hunk 2]
```diff
@@ -7,13 +7,13 @@ echo "usage: $0 architecture"
 else
 
 TARGET=$1
-V=2.25
+V=2.34
 
 cd /tmp
-wget -nc http://ftp.gnu.org/gnu/binutils/binutils-$V.tar.gz
-wget -nc http://ftp.gnu.org/gnu/binutils/binutils-$V.tar.gz.sig
+wget -nc https://ftp.gnu.org/gnu/binutils/binutils-$V.tar.gz
```
Comment (Arusekk) [inline generic/install.sh:13]: Why do this? If we verify the signature later, this encryption seems redundant

File: macos/binutils-aarch64.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 class BinutilsAarch64 < Formula
-  homepage "http://www.gnu.org/software/binutils/binutils.html"
-  url "http://ftpmirror.gnu.org/binutils/binutils-2.25.tar.gz"
-  mirror "http://ftp.gnu.org/gnu/binutils/binutils-2.25.tar.gz"
-  sha256 "cccf377168b41a52a76f46df18feb8f7285654b3c1bd69fc8265cb0fc6902f2d"
+  homepage "https://www.gnu.org/software/binutils/binutils.html"
+  url "https://ftpmirror.gnu.org/binutils/binutils-2.34.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/binutils/binutils-2.34.tar.gz"
+  sha256 "53537d334820be13eeb8acb326d01c7c81418772d626715c7ae927a7d401cab3"
```
Comment (Arusekk) [inline macos/binutils-aarch64.rb:5]: Again, if we verify a checksum, why use encrypted transmission?

File: macos/generate.sh [hunk 4]
```diff
@@ -3,7 +3,7 @@ source ../common/arch.sh
 
 title_case()
 {
-    python -c 'import sys; print " ".join(map(str.title, sys.argv[1:]))' $*
+    python -c 'import sys;import re; print " ".join(map(str.capitalize, [sys.argv[1].replace("_", "")]))' $*
```
Comment (Arusekk) [inline macos/generate.sh]: Could this be python3-compatible? I don't know for how long macOS keeps python2 in the future.
```suggestion
    python -c 'import sys;import re; print(" ".join(map(str.capitalize, [sys.argv[1].replace("_", "")])))' $*
```

File: macos/generate.sh [hunk 5]
```diff
@@ -12,10 +12,10 @@ file=binutils-$arch.rb
...
 cat > $file <<EOF
 class Binutils$(title_case $arch) < Formula
-  homepage "http://www.gnu.org/software/binutils/binutils.html"
-  url "http://ftpmirror.gnu.org/binutils/binutils-2.25.tar.gz"
-  mirror "http://ftp.gnu.org/gnu/binutils/binutils-2.25.tar.gz"
-  sha1 "f10c64e92d9c72ee428df3feaf349c4ecb2493bd"
+  homepage "https://www.gnu.org/software/binutils/binutils.html"
+  url "https://ftpmirror.gnu.org/binutils/binutils-2.34.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/binutils/binutils-2.34.tar.gz"
+  sha256 "53537d334820be13eeb8acb326d01c7c81418772d626715c7ae927a7d401cab3"
```
Comment (Arusekk) [inline macos/generate.sh:18]: Checksum verification seems enough to me, why use https for download? (I approve https homepage, though)

File: macos/install_all.sh (hunk 1 above)
Comment (Arusekk) [inline macos/install_all.sh]: Have you tested this? Someone pointed out that this might be wrong.

**Review by Arusekk — CHANGES_REQUESTED** [review]

File: macos/generate.sh (hunk 5 above)
Comment (eljeffeg) [reply macos/generate.sh:18]: I just did a search replace for all http.  It doesn't even look like I can use this on mac due to the qemu.  So.. however you want to do it - I was just trying to keep it up to date.

File: macos/install_all.sh (hunk 1 above)
Comment (eljeffeg) [reply macos/install_all.sh]: Must have accidentally saved the install_all.sh when I was closing it out for the update_all.sh

File: generic/install.sh (hunk 2 above)
Comment (eljeffeg) [reply generic/install.sh:13]: Just updating what was there.. feel free to change it.

File: macos/generate.sh (hunk 5 above)
Comment (Arusekk) [reply macos/generate.sh:18]: Alright, whatever, I am too busy lately to decide stuff, so I'll just go with this.

**Pull request merged**